### DISCUSSION TITLE
Add a "Disable Web Search" option for the start menu

### DISCUSSION THREAD

**Discussion by cgriffiths86** [ChrisTitusTech/winutil#348] [Ideas]
On any Windows install I use the tweak to disable the web search in the start menu as I prefer the search just looking on my machine. Also, because without using 3rd party software you can't have it use a different browser/search engine than Bing/Edge. I'd love for it to be one of the options in the "tweaks" section to make your tool a one-stop shop.

**Comment by DeveloperDurp** [comment]
Once #576 is merged I will investigate and implement if possible.

https://www.kapilarya.com/how-to-disable-web-search-for-start-menu-in-windows-10 

This looks to disable web search for me
HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Search\BingSearchEnabled = 0

**Comment by gmy77** [comment]
Or Disable the search feature of Windows at all... the index search service. Thing i disable manually any time i have to install new windows.
